README.md

```diff
@@ -35,14 +35,14 @@ pyenv global system 3.8.10
 
 # Installation
 
-Create and activate a virtual environment.
+1. Create and activate a virtual environment.
 
 ```shell
```

Comment:
Please modify the readme with a couple of spaces to each line so that the content under  a list flows appropriately.  Currently the content is left justified rather than under the 1.  This also means you cant use #. to autocreate the list for you.

Reply:
I meant so it looks like
![image](https://user-images.githubusercontent.com/3979063/210017186-cd778cbd-829f-4436-9c8c-679ab9ec54ba.png)

I committed an example for the 1st item.  You should be able to use 1. all the way down if you do the spaces in the file correctly.

Please contact me if this doesn't make sense.

Reply:
For ordered lists to properly automformat using '1.', I had to ensure that all text under each numbered item was indented.

Reply:
Yes that's right.